### PR TITLE
Feature/create room

### PR DESCRIPTION
**Pull request by VVMig**

### CODE REVIEW

File: src/app/create-room/CreateRoom.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  const [createRoom, { data: roomData, loading: roomLoading }] =
+    useMutation<CreatRoomQuery>(CREATE_ROOM);
+
+  const { data: rangeData, loading: rangeLoading } =
+    useQuery<GetRangeParticipantsQuery>(GET_RANGE_PARTICIPANTS);
+
+  const [maxParticipants, setMaxParticipants] = useState<null | number>(null);
+
+  const onChangeRange: React.FormEventHandler<HTMLInputElement> = (event) => {
+    setMaxParticipants(Number(event.currentTarget.value));
```
Comment (RuslanShohanov) [inline src/app/create-room/CreateRoom.tsx]: Why do we need that Number cast? What values could be in currentTarget.value except of numbes?

File: src/app/layout/header/SwitchLang.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import { useOutsideClick } from '../../hooks';
+import { Styled } from './styled';
+
+interface Props {
+  isLangActive: boolean;
+  onToggleLang: () => void;
+}
+
+export const SwitchLang: React.FC<Props> = ({ isLangActive, onToggleLang }) => {
+  const switchLangRef = useRef<HTMLDivElement | null>(null);
```
Comment (RuslanShohanov) [inline src/app/layout/header/SwitchLang.tsx]: Lang is Language? It's better to use whole word in such cases

File: src/app/layout/header/SwitchLang.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  useOutsideClick(switchLangRef, () => {
+    if (isLangActive) {
+      onToggleLang();
+    }
+  });
+
+  return (
+    <Styled.SwitchLangWrapper ref={switchLangRef}>
+      <Styled.SwitchLang isActive={isLangActive} onClick={onToggleLang}>
+        En
```
Comment (RuslanShohanov) [inline src/app/layout/header/SwitchLang.tsx]: I'd extract language values into a separate enum

File: src/app/query/mutation.ts [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import { gql } from '@apollo/client';
+
+export const CREATE_ROOM = gql`
+  mutation CreateRoom($maxParticipants: String) {
```
Comment (RuslanShohanov) [inline src/app/query/mutation.ts]: Why is it String? Not a number?

File: src/packages/components/spinner/Spinner.tsx [hunk 5]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import { Styled } from './styled';
+
+export const Spinner = () => {
+  return (
+    <Styled.Spinner>
+      <div></div>
```
Comment (RuslanShohanov) [inline src/packages/components/spinner/Spinner.tsx:8]: Are these elements temporary?

File: src/app/create-room/CreateRoom.tsx (hunk 1 above)
Comment (VVMig) [reply src/app/create-room/CreateRoom.tsx]: Input values is always string
But i remade with + convert

File: src/app/layout/header/SwitchLang.tsx (hunk 3 above)
Comment (VVMig) [reply src/app/layout/header/SwitchLang.tsx]: Yes, i will
Now is only as a mock data
i will use react-i18next library
And my translate words will be strored in public folder

File: src/packages/components/spinner/Spinner.tsx (hunk 5 above)
Comment (VVMig) [reply src/packages/components/spinner/Spinner.tsx:8]: No

**Review by RuslanShohanov — APPROVED** [review]

**Pull request merged**